web/src/components/App.vue

```diff
@@ -2948,16 +2950,27 @@ export default {
                     }
                 }
             }
+
+            // Format Name Strnig
             let newNameBase = this.eventExtras[this.selectedEvent].video_prefix + ' ' + prefix + ' Match ' + matchNumber;
             let newName = newNameBase;
+
+            // If this isn't the first play
             if (videoNames.includes(newNameBase + extension) && matchPlay[1] > 1) {
                 newNameBase += ' Play ' + matchPlay[1];
                 newName = newNameBase;
             }
+
+            // If we already have a video with this name, add " - n" to it
             for (let i = 2; videoNames.includes(newName + extension); i++) {
                 newName = newNameBase + ' - ' + i;
             }
 
+            // If we have a match state, and it is e-stopped
+            if(typeof matchState !== "undefined" && matchState ===  FIELD_STATE.MatchCancelled) {
+                newName += ' - Field Fault'
+            }
+
```

Comment:
Looks like I forgot to leave a comment. This should go above the deduplication logic (the `for` loop above) - otherwise we could end up with a name conflict. For instance, if we have `Match 3 - Field Fault.mp4` already, and no other files starting with `Match 3 -`, we would end up assigning `Match 3 - Field Fault` to `newName` here.

Also, just checking for `matchState === FIELD_STATE.MatchCancelled` is enough, since that also checks that matchState isn't `undefined`.